recipe/meta.yaml

```
@@ -25,13 +25,16 @@ source:
   #   folder: numpy/core/src/npysort/x86-simd-sort
 
 build:
-  number: 0
+  number: 1
   skip: true  # [py<39]
   entry_points:
     - f2py = numpy.f2py.f2py2e:main             # [win]
     - numpy-config = numpy._configtool:main     # [win]
   run_exports:
-    - {{ pin_subpackage("numpy") }}
+    # numpy will by default use the ABI feature level for the first numpy version
+    # that added support for the oldest currently-supported CPython version; see
+    # https://github.com/numpy/numpy/blob/v2.0.0rc1/numpy/_core/include/numpy/numpyconfig.h#L124
+    - {{ pin_subpackage("numpy", min_pin="1.19") }}
```

Think we want `lower_bound` when using a version number (instead of `x.x` syntax)

```suggestion
    - {{ pin_subpackage("numpy", lower_bound="1.19") }}
```

Also we should double that `lower_bound` can be used with the default `max_pin="x"`. If not, we might want `upper_bound="3.0.0a0"

ref: https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#extra-jinja2-functions

---

No you're right, this should be `lower_bound`. Ugh, I actually looked at the docs, but keep forgetting about the placeholder ("x.x.") vs. actual ("1.19") distinction...

---

Wrong again... 😑 only `pin_compatible` has a `lower_bound=` kwarg, and the linter won't let us use that because `numpy` is a known output here.

---

So I looked at the implementation of `pin_compatible`, and it's just not build for this scenario - it's too unusual to use a lower bound that's below the major version used at build time.

---

Interesting should we raise a conda-build issue?

---

IMO no, the case is too exceptional for this API to make sense.